openapi/ga4gh-tool-discovery.yaml

```diff
@@ -38,8 +38,8 @@ paths:
           required: true
           type: string
           description: >-
-            A unique identifier of the tool, scoped to this registry, for
-            example `123456`.
+            A unique identifier of the tool, scoped to this registry, with a CURIE-like prefix for
```

Comment:
I think the convention with `for example` is to have commas before and after. But you'll end up with a slightly unwieldy sentence.

Reply:
`x-example` property also works.  It should get automatically converted to `example` in openapi